### PR TITLE
update-titles

### PR DESCRIPTION
**Pull request by paulchrisluke**

### CODE REVIEW

**Comment by mort3za** [comment]
@paulchrisluke please change base branch to staging

**Comment by paulchrisluke** [comment]
done @mort3za

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "volkswagen-email-cms",
+  "name": "SendBlobby - Corperate Communications Simplified",
```
Comment (mort3za) [inline package.json:2]: I'm not sure that you can use space and uppercase characters in package name

**Comment by mort3za** [comment]
will create a new PR.

**Pull request closed without merging**